crates/sui-types/src/committee.rs

```diff
@@ -134,12 +134,15 @@ impl Committee {
     }
 
     pub fn public_key(&self, authority: &AuthorityName) -> SuiResult<AuthorityPublicKey> {
+        debug_assert_eq!(self.expanded_keys.len(), self.voting_rights.len());
```

Comment:
It's not always guaranteed that this is loaded, right?

Reply:
I see that `Committee::new` initializes expanded_keys... Do you mean it might not be initialized if Committee is deserialized? Or is there any code path where this can happen?

Reply:
Right it may not be initialized if it's deserialized. I can imagine that this will happen whenever we load committee from the committee store

Reply:
Good catch. Adding code to initialize this on the deserialization